src/templates/finops-hub/modules/infrastructure.bicep

```diff
@@ -303,7 +303,7 @@ resource scriptStorageAccount 'Microsoft.Storage/storageAccounts@2022-09-01' = i
       defaultAction: 'Deny'
```

Comment:
Add a dependency for `vNet::scriptSubnet` rather than using an explicit reference